backend/spherre/app/service/transaction.py

```
@@ -0,0 +1,297 @@
+from typing import List, Optional
+
+from spherre.app.models import session_save
+from spherre.app.models.account import Account, Member
+from spherre.app.models.transaction import Transaction, TransactionStatus, TransactionType
+
+
+class TransactionService:
+    @classmethod
+    def create_transaction(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)